src/state/items.ts

```diff
@@ -1,30 +1,42 @@
-import * as mintabseAapi from 'mintbase'
 import { MintMetadata } from 'mintbase'
+import { formatNearAmount } from 'near-api-js/lib/utils/format'
 import { selectorFamily } from 'recoil'
 import urlcat from 'urlcat'
 import { IWallet, mintbaseContract, network } from './near'
 
-export interface ItemWithMetadata {
-  id: string
-  minter: string
-  tokens: Token[]
-  thing: MintMetadata
+export interface ThingWithMetadata {
+  thing: StoreThing
+  metadata: MintMetadata
 }
 
-export interface Token extends mintabseAapi.Token {
+// mintabseAapi.Token
+export type Token = {
+  id: string
+  thingsId: string
   minter: string
+  ownerId: string
+  store: {
+    baseUri: string
+  }
+  list: {
+    price: number
+    acceptedOfferId: string
+  } | null
 }
 
-export interface StoreThings {
+export type StoreThing = {
   id: string
-  tokens: Token[]
-}
-
-interface ExtendedStoreThings extends StoreThings {
   metaId: string
   store: {
     baseUri: string
   }
+  tokens: Token[]
+}
+
+export enum ThingStatus {
+  minted = 'MINTED',
```

Comment:
Let's have only Listed / Not Listed status. Because there cannot be not minted item, and after the item is transferred to another account it can be listed again.

Reply:
Still need a third status, when you have minted and sold an item. As you can't burn it anymore

Reply:
You should burn only owned items, not minted